src/python/projenrc.ts

```diff
@@ -22,6 +22,12 @@ export interface ProjenrcOptions {
    * @default - current version
    */
   readonly projenVersion?: string;
+
+  /**
+   * Python executable to use. default python3 alternative:  python.
+   * @default  "python3"
```

Comment:
My impression is that this is still the most portable default. Also if we change that it might be annoying for some.
```suggestion
   * @default  "python"
```